spec/services/teacher_training_api/import_course_spec.rb

```diff
@@ -40,6 +57,11 @@ module TeacherTrainingApi
           it "does not creates a duplicate course" do
             expect { subject }.not_to(change { Course.count })
           end
+
+          it "does not create a duplicate course_subject join" do
+            subject
+            expect { subject }.not_to(change { CourseSubject.count })
+          end
         end
 
         context "with a different name" do
```

Comment:
If name can change needs a test to show:

- subjects will not be duplicated
- subjects can be updated